packages/account/src/Components/poi/idv-document-submit/__tests__/idv-document-submit.spec.tsx

```diff
@@ -126,25 +134,27 @@ describe('<IdvDocumentSubmit/>', () => {
         expect(document_number_input.name).toBe('document_number');
         expect(document_number_input).toBeDisabled();
 
-        fireEvent.change(document_type_input, { target: { value: 'Test document 2 name' } });
-        expect(document_number_input).toBeEnabled();
+        userEvent.selectOptions(document_type_input, 'Test document 2 name');
+        await waitFor(() => {
+            expect(document_number_input).toBeEnabled();
+        });
         expect(screen.queryByText(/please enter the correct format/i)).not.toBeInTheDocument();
-
+        (isDocumentNumberValid as jest.Mock).mockReturnValueOnce('please enter your document number');
         fireEvent.blur(document_number_input);
         expect(await screen.findByText(/please enter your document number/i)).toBeInTheDocument();
 
-        fireEvent.keyUp(document_number_input);
+        (isDocumentNumberValid as jest.Mock).mockReturnValueOnce('please enter the correct format');
         fireEvent.change(document_number_input, { target: { value: 'A-32523' } });
```

Comment:
`userEvent.type(document_number_input, 'A-32523' )`

Reply:
testcase fails when userEvent.type is used

Reply:
same as above